File: pyiArduinoI2Cph/examples/getFunction.py
# ПРИМЕР ПОЛУЧЕНИЯ И ВЫЧИСЛЕНИЯ ДАННЫХ:

from time import sleep

#   Подключаем библиотеку   для работы с pH-метром I2C-flash.
from pyiArduinoI2Cph import *

#   Объявляем объект sensor для работы с функциями и методами библиотеки pyiArduinoI2Cph, указывая адрес модуля на шине I2C.
sensor = pyiArduinoI2Cph(0x09)

while True:
     print("Кислотность = pHn+"      , end='')
     print("( Vn -Vout)/( Vstep *Ky)", end='')
     print("\r\n            = "      , end='')
     print(sensor.getPHn()           , end='')
     print("+("                      , end='')
     print(sensor.getVn()            , end='')
     print("-"                       , end='')
     print(sensor.getVout()          , end='')
     print(")/("                     , end='')
     print(sensor.getVstp()/1000     , end='')
     print("* "                      , end='')
     print(sensor.getKy()            , end='')
     print(") = "                    , end='')
     print(sensor.getPH()            , end='')
     print(" pH.\r\n"                , end='')
     sleep(1)

File: pyiArduinoI2Cph/cpp/getFunction.cpp
// ПРИМЕР ПОЛУЧЕНИЯ И ВЫЧИСЛЕНИЯ ДАННЫХ:          //
#include "Serial.h"
                                                  //
#include "../iarduino_I2C_pH.h"                   //   Подключаем библиотеку   для работы с pH-метром I2C-flash.
iarduino_I2C_pH sensor(0x09);                     //   Объявляем объект sensor для работы с функциями и методами библиотеки iarduino_I2C_pH, указывая адрес модуля на шине I2C.
                                                  //   Если объявить объект без указания адреса (iarduino_I2C_pH sensor;), то адрес будет найден автоматически.
void setup(){                                     //
     sensor.begin();                              //   Инициируем работу с pH-метром I2C-flash.
}                                                 //
                                                  //
void loop(){                                      //
     Serial.print("Кислотность = pHn+"      );    //
     Serial.print("( Vn -Vout)/( Vstep *Ky)");    //
     Serial.print("\r\n            = "      );    //
     Serial.print(sensor.getPHn()        , 1);    //
     Serial.print("+("                      );    //
     Serial.print(sensor.getVn()         , 2);    //
     Serial.print("-"                       );    //
     Serial.print(sensor.getVout()       , 2);    //
     Serial.print(")/("                     );    //
     Serial.print(sensor.getVstp()/1000  , 5);    //
     Serial.print("* "                      );    //
     Serial.print(sensor.getKy()         , 0);    //
     Serial.print(") = "                    );    //
     Serial.print(sensor.getPH()         , 1);    //
     Serial.print(" pH.\r\n"                );    //
     delay(1000);                                 //
}                                                 //

int main()
{
	setup();
	for (;;)
		loop();
}
